### PR TITLE
Add "Position" to logger.event and logger.error (PROTO-21)

### PR DESCRIPTION
**Pull request by nabhanelrahman**

### CODE REVIEW

File: logger/error.proto [hunk 1]
```diff
@@ -92,6 +92,15 @@ message Error {
     AD_SERVER = 1200;
   }
 
+  enum Position {
+    NOT_KNOWN = 0;
```
Comment (kalbasit) [inline logger/error.proto]: let's keep it `UNKNOWN` to be consistent with the rest of the enums

File: logger/error.proto [hunk 2]
```diff
@@ -92,6 +92,15 @@ message Error {
...
+
+    INSTREAM = 100;
+    PREROLL = 101;
```
Comment (kalbasit) [inline logger/error.proto]: align this with `:EasyAlign` :)

File: logger/event.proto [hunk 3]
```diff
@@ -49,6 +49,15 @@ message Event {
     BREAK_END                = 419;
   }
 
+  enum Position {
```
Comment (kalbasit) [inline logger/event.proto]: this and the enum above are the same, please move it to a different file (same as subscription.proto).

File: logger/error.proto (hunk 1 above)
Comment (nabhanelrahman) [inline logger/error.proto]: I get this error `event.proto:53:5: "UNKNOWN" is already defined in "logger.Event".
event.proto:53:5: Note that enum values use C++ scoping rules, meaning that enum values are siblings of their type, not children of it.  Therefore, "UNKNOWN" must be unique within "logger.Event", not just within "Position".
`

File: logger/position.proto [hunk 4]
```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+package logger;
+
+enum Position {
+  NOT_KNOWN = 0;
```
Comment (kalbasit) [inline logger/position.proto]: `s/NOT_KNOWN/UNKNOWN` to stay consistent with the enums

File: logger/position.proto [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+  INSTREAM = 100;
+  PREROLL = 101;
```
Comment (kalbasit) [inline logger/position.proto]: EasyAlign :)

**Pull request merged**